t/19-isotovideo-command-processing.t

```diff
@@ -6,6 +6,7 @@ use Test::More;
 use Test::MockModule;
 use Test::Warnings;
 use OpenQA::Isotovideo::CommandHandler;
+use bmwqemu;
```

Comment:
Wouldn't all the changes fit into another test module better when we now actually require bmwqemu?

Reply:
This test module is about command processing of isotovideo and the new tests are exactly about that.

Some commands might use `bmwqemu` internally e. g. to access the backend. So I'm using it here to assign a fake backend. That shouldn't be a problem, right?

Reply:
yeah, sure. But if we need to override bmwqemu and in former tests we didn't it feels like we conduct tests on different levels where I would suggest to move to other test files. But with this stated I leave the decision to you and trust you.

Reply:
I thought about it and added another commit to get rid of `use bmwqemu` as it is really only used for mocking the backend but not by itself. This hopefully clarifies the code.